daml-lf/spec/daml-lf-1.rst

```diff
@@ -2874,7 +2874,6 @@ as described by the ledger model::
    —————————————————————————————————————————————————————————————————————— EvUpdCreateErr4
      'create' @Mod:T vₜ ‖ (st₀, keys₀)  ⇓ᵤ  (Err E, ε)
 
-
```

Comment:
?

Reply:
On that note, we probably should add it to the spec but doesn’t have too be in this PR.